config/environments/production.rb

```diff
@@ -25,7 +25,7 @@
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier
```

Comment:
Aren't there any none-js compressors that do not screw the build time?

Reply:
I couldn't find an alternative javascript compressor to uglifier. It is used by default when using ruby on rails.

Javascript files are only sent when a user navigates to the consumer documentation. And in that case, most of the javascript is already minified.
The consumer's core tasks (buying products and providing an API) don't use javascript.
That's why I think it's ok to not use a javascript compressor.

Reply:
Fair enough.

Reply:
Nice catch with that JS runtime.